### ISSUE TITLE
Problem with roms

### ISSUE THREAD

**Issue by goldenking277** [SachinVin/citra_android#88]
I dont know why but whenever i would get a rom and out it in it shows up as a black and grey x (for the games picture) and wont load at all i dont know if im using the wrong type of rom file or not. (The files always end in .3ds)

**Comment by Priyam005** [comment]
Did you decrypt for roms??

**Comment by goldenking277** [comment]
Do they have to be decrypt to work?

**Comment by HelloOO7** [comment]
Yes. GodMode9 should have done that for you though. How did you dump the games?

**Comment by goldenking277** [comment]
@hello007 you mean like get the files, or how i put them into the emulator.

**Comment by HelloOO7** [comment]
The first option. Most 3DS software dumpers already decrypt the game for you, but who knows, maybe you used some ancient one without decryption.

**Comment by goldenking277** [comment]
@Hello007 so is that the problem im using the wrong files?  The ones i mostly went for were with out the decrypted. So would that be my problem?

**Comment by HelloOO7** [comment]
If they weren't decrypted, then they are not going to work. So yes, you need to redump them decrypted.

**Comment by goldenking277** [comment]
@hello007 thank you. I didnt know what the problem was so thank you.

**Comment by Guns29** [comment]
the roms wont load until you decrypted them until you do this the screen will stay black with encrypted roms thats why
